test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/delegatingclients/DelegatingSyncClientTest.java

```diff
@@ -49,6 +51,8 @@ public class DelegatingSyncClientTest {
 
     MockSyncHttpClient mockSyncHttpClient = new MockSyncHttpClient();
     ProtocolRestJsonClient defaultClient = ProtocolRestJsonClient.builder()
+                                                                 .credentialsProvider(AnonymousCredentialsProvider.create())
+                                                                 .region(Region.US_WEST_2)
```

Comment:
Fixed tests when credentials and region aren't configured. Unrelated to this change.